trl/trainer/dpo_trainer.py

```diff
@@ -164,6 +180,7 @@ def __init__(
                 )
                 max_target_length = 128
 
+            self.max_length = max_length
```

Comment:
Do you know why before this PR we were using `self.config.max_length`?

Reply:
It seems fine to stick with `self.config.max_length` imo.

Reply:
the function with `self.config.max_length` was not used, and `self.config.max_length` did not exist.
I think maybe it should be `self.args.max_length`, let me check.

Reply:
I double checked, `max_length` is not in `TrainerArguments`, so it needs to be handled in the `DPOTrainer` init